Sloth.Core/Services/AggieEnterpriseService.cs

```diff
@@ -156,8 +156,8 @@ public async Task<IGlJournalRequestResult> CreateJournal(Source source, Transact
         /// <param name="requestId">Found in JournalRequest.RequestId</param>
         /// <returns></returns>
         public async Task<IGlJournalRequestStatusResult> GetJournalStatus(string requestId)
```

Comment:
would a better solution be to just take a `Guid` as a param?

Reply:
Less things to change... I think we are just storing it as a string. At least I was in OPP.

Reply:
NM, it is a guid. I'll fix